sdk/synapse/azure-analytics-synapse-accesscontrol/src/main/java/com/azure/analytics/synapse/accesscontrol/AccessControlClient.java

```diff
@@ -0,0 +1,134 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
All the API's need to add support [context propagation](https://azure.github.io/azure-sdk/java_design.html#common-service-client-patterns).

Reply:
Fixed